{"name":"Story 991, Test 1205 ; ValidateCallTakeCantCancelCaseWithoutCancellationReason","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1688979883711,"stop":1688979944976,"uuid":"1eaabba0-93f7-4b92-95b1-4c8ecc5c6de5","historyId":"c277972243833ab5285fe82598a6501","fullName":"Execute.CallTakerExecute.ValidateCallTakeCantCancelCaseWithoutCancellationReason","labels":[{"name":"package","value":"Execute.CallTakerExecute"},{"name":"testClass","value":"Execute.CallTakerExecute"},{"name":"testMethod","value":"ValidateCallTakeCantCancelCaseWithoutCancellationReason"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"BCAA"},{"name":"subSuite","value":"Execute.CallTakerExecute"},{"name":"host","value":"LAP10-GUYL"},{"name":"thread","value":"28508@LAP10-GUYL.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}